Formula/kontena-cli.rb

```diff
@@ -0,0 +1,202 @@
+    end
+
+    resource "tty-screen" do
+      url "https://rubygems.org/gems/tty-screen-0.5.0.gem"
+      sha256 "c33a2d1d0b857cd37515a6db6fd5c6dd05e04f017b92753d3f31e54013bdb900"
+    end
+
+    resource "tty-table" do
+      url "https://rubygems.org/gems/tty-table-0.8.0.gem"
+      sha256 "87d81d2d1780de5beb8c25848b027ddcdec7fb407d766aafde0c4f032ccf36c8"
+    end
+
+    resource "unicode-display_width" do
+      url "https://rubygems.org/gems/unicode-display_width-1.1.3.gem"
+      sha256 "1d6247805b2431fd3d27614f6e0c38665b068bfc3a58cf89f08b7f107daf48e4"
+    end
+
+    resource "unicode_utils" do
+      url "https://rubygems.org/gems/unicode_utils-1.4.0.gem"
+      sha256 "b922d0cf2313b6b7136ada6645ce7154ffc86418ca07d53b058efe9eb72f2a40"
+    end
+
+    resource "verse" do
+      url "https://rubygems.org/gems/verse-0.5.0.gem"
+      sha256 "4c8af6322b91e115a772af1acdf99952e391ad14890577674e11e52d474af2e0"
+    end
+
+    resource "websocket-driver" do
+      url "https://rubygems.org/gems/websocket-driver-0.6.5.gem"
+      sha256 "b4a4e0c75eae9e85ceb248d4dc13519da581452767dcd5126c1452ea052183a4"
+    end
+
+    resource "websocket-extensions" do
+      url "https://rubygems.org/gems/websocket-extensions-0.1.2.gem"
+      sha256 "7919c0310edda55ce4026d3f160e612160493c048276cabe500308439dd2161f"
+    end
+
+    resource "wisper" do
+      url "https://rubygems.org/gems/wisper-2.0.0.gem"
+      sha256 "6d8623c76943612e8db81862af7eb20f5a8ad631fe1857d9f7ced97e87e5222b"
+    end
+
+    resource "kontena-plugin-cloud" do
+      url "https://rubygems.org/gems/kontena-plugin-cloud-1.1.0.gem"
+      sha256 "87e88329c17c6aa6c49069fd6fa5f6577c451f73bb2bf264051b858af8ea57f1"
+    end
+  end
+
+  depends_on :ruby => "2.1"
+
+  def install
+    ENV["GEM_HOME"] = libexec
+
+    cd "cli" do
+      system "gem", "build", "--norc", "kontena-cli.gemspec"
+
+      if build.head?
+        system "gem", "install", Dir["kontena-cli-*.gem"].first,
+               "--no-document", "--norc", "--install-dir", libexec
+      else
+        resources.each do |r|
+          r.verify_download_integrity(r.fetch)
+          system "gem", "install", r.cached_download, "--ignore-dependencies",
+                 "--no-document", "--norc", "--install-dir", libexec
+        end
+
+        system "gem", "install", "--ignore-dependencies", "--norc",
+               "--no-document", "kontena-cli-#{version}.gem"
+      end
+    end
+
+    bin.env_script_all_files(libexec/"bin", :GEM_HOME => ENV["GEM_HOME"])
+
+    env = { :GEM_PATH => libexec, :KONTENA_EXTRA_BUILDTAGS => "homebrew" }
+    env[:KONTENA_EXTRA_BUILDTAGS].concat ",head" if build.head?
+    (bin/"kontena").write_env_script(libexec/"bin/kontena", env)
+
+    zsh_completion.install "cli/lib/kontena/scripts/kontena.zsh" => "_kontena"
+    bash_completion.install "cli/lib/kontena/scripts/kontena.bash" => "kontena"
+  end
+
+  test do
+    output = shell_output("#{bin}/kontena --version")
```

Comment:
I'd leave out the intermediate variable for anything that doesn't go passed the eightieth column. So probably
```
  test do
    assert_match "+homebrew", shell_output("#{bin}/kontena --version")
    assert_match "login", shell_output("#{bin}/kontena complete kontena master")
    output = shell_output("#{bin}/kontena plugin search digitalocean")
    assert_match "Kontena DigitalOcean plugin", output
    output = shell_output("#{bin}/kontena stack reg show kontena/hello-world")
    assert_match "description: Sample stack to test Kontena", output
    assert_match "NAME", shell_output("#{bin}/kontena master list")
  end
```

Reply:
Yep, went for uniformity.

Reply:
The second assert has ")" on column 80 in your example, adding a linefeed makes it go to 81.

This would change one assert only.

Reply:
The linefeed doesn't count in terms of columns.